src/main/java/network/brightspots/rcv/StreamingCvrReader.java

```diff
@@ -186,7 +186,11 @@ private void endCvr() {
 
     if (idColumnIndex != null && currentSuppliedCvrId == null) {
       Logger.log(
-          Level.SEVERE, "Cast vote record identifier not found for: %s", computedCastVoteRecordId);
+          Level.SEVERE,
+          "Cast vote record identifier missing on row %d in file %s.  This may be due to an "
```

Comment:
Recommend copying in the below because nits:
```
"Cast vote record identifier missing in row %d in file:\n%s\n\nThis may be due to an incorrectly formatted *.xlsx file. Try copying your CVR data into a new *.xlsx file to fix this.",
```

Reply:
Eh

Reply:
grunted the anti-pedantic Canadian